backend/lib/averages.ts

```diff
@@ -24,7 +24,13 @@ export const averageTasks = (evaluations: Evaluation[]) => {
  */
 export const averageRating = (evaluations: Evaluation[]) => {
   const total = evaluations.length;
-  const sum = 100; // TODO: Task 3 - backend - Somma i voti di tutte le valutazioni
+  if (total === 0) return "0"; // Gestisce il caso in cui non ci siano valutazioni
+
+  // Somma i voti di tutte le valutazioni
+  const sum = evaluations
+    .map((evaluation) => evaluation.rating)
```

Comment:
Sarebbe ok, ma non esiste la proprietà "rating" in questo oggetto, la proprietà da valutare è "valutazione"